src/app/phast/losses/wall-losses/wall-losses-form/wall-losses-form.component.ts

```diff
@@ -91,17 +89,10 @@ export class WallLossesFormComponent implements OnInit {
   focusOut() {
     this.changeField.emit('default');
   }
-  //check inputs for errors
-  checkWarnings() {
-    let tmpLoss: WallLoss = this.wallLossesService.getWallLossFromForm(this.wallLossesForm);
-    this.warnings = this.wallLossesService.checkWarnings(tmpLoss);
-    let hasWarning: boolean = this.wallLossesService.checkWarningsExist(this.warnings);
-    this.inputError.emit(hasWarning);
-  }
 
   //on input/change in form startSavePolling is called, if not called again with 3 seconds save process is triggered
```

Comment:
Can you please delete this comment about startSavePolling. So old and not accurate anymore. Thanks